### ISSUE TITLE
[Question] Containerd and HTTPS

### ISSUE THREAD

**Issue by hyy0322** [duyanghao/Eagle#3]
1. Does this project support containerd ?
2. Does this project support https registry ?

**Comment by duyanghao** [comment]
> 1. Does this project support containerd ?

Eagle doesn't support containerd currently since it aims to support docker natively, but it can be acheived easily by extending this project.

> 2. Does this project support https registry ?

Again, https protocol support can be acheived by extending `getDataFromOrigin` function of seeder.

Welcome your PR if it's urgent : )

**Comment by liyimeng** [comment]
is this project still alive?

**Comment by duyanghao** [comment]
> is this project still alive?

Yeah, I will continue to maintain it using my spare time since it's a personal repository.

**Comment by liyimeng** [comment]
@duyanghao thanks for update and thanks for the great work!
